<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Client</class>
 <widget class="QMainWindow" name="Client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>724</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="connectBox">
        <property name="title">
         <string>连接</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="hostLabel">
               <property name="text">
                <string>IP/Domain: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="hostEditor">
               <property name="text">
                <string>182.92.88.252</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="portLabel">
               <property name="text">
                <string>Port: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="portEditor">
               <property name="text">
                <string>21</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPushButton" name="connectButton">
               <property name="text">
                <string>Connect</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="quitButton">
               <property name="text">
                <string>Quit</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="loginBox">
        <property name="title">
         <string>登录</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="userLabel">
               <property name="text">
                <string>User:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="userEditor">
               <property name="text">
                <string>webstorm</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="passwordLabel">
               <property name="text">
                <string>Password: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="passwordEditor">
               <property name="text">
                <string>lyh26239231</string>
               </property>
               <property name="echoMode">
                <enum>QLineEdit::Password</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QPushButton" name="loginButton">
               <property name="text">
                <string>Login</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="modeBox">
               <property name="text">
                <string>PORT</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="fileBox">
        <property name="title">
         <string>文件管理</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QPushButton" name="refreshButton">
             <property name="text">
              <string>刷新</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="newButton">
             <property name="text">
              <string>新建</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="deleteButton">
             <property name="text">
              <string>删除</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="downloadButton">
             <property name="text">
              <string>下载</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="uploadButton">
             <property name="text">
              <string>上传</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="renameButton">
             <property name="text">
              <string>重命名</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLineEdit" name="pwdEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableView" name="fileView">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="gridStyle">
            <enum>Qt::NoPen</enum>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>25</number>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="cmdBox">
        <property name="title">
         <string>命令行提示</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="cmdLabel">
           <property name="text">
            <string>Command Prompt: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="cmdLine">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="html">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; line-height:9px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>814</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>hostEditor</tabstop>
  <tabstop>portEditor</tabstop>
  <tabstop>connectButton</tabstop>
  <tabstop>quitButton</tabstop>
  <tabstop>userEditor</tabstop>
  <tabstop>passwordEditor</tabstop>
  <tabstop>loginButton</tabstop>
  <tabstop>cmdLine</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
